public/_locales/en/messages.json

```diff
@@ -22,5 +22,29 @@
     },
     "saveSuccessful": {
         "message": "Page was saved successfully."
+    },
+    "optionsGeneralCardTitle": {
+        "message": "General"
+    },
+    "optionsIsLoadInNewTab": {
+        "message": "Load WebMark in current tab"
+    },
+    "optionsAboutCardTitle": {
+        "message": "About"
+    },
+    "optionsAboutCardContentVersionTitle": {
+        "message": "Current version"
+    },
+    "optionsAboutCardContentTeamNameTitle": {
+        "message": "Author"
+    },
+    "optionsAboutCardContentTeamNameContent": {
+        "message": "strgzr"
+    },
+    "optionsAboutCardContentTeamLinkTitle": {
+        "message": "Homepage"
```

Comment:
Is this a good word? I can't think of a better word, but I feel like this is often used in Konglish.